Dockerfile

```diff
@@ -1,6 +1,20 @@
 FROM ubuntu:20.04
 
-RUN apt-get update && apt-get install -y wget gcc-arm-linux-gnueabi gcc-arm-linux-gnueabihf git build-essential bison flex bc u-boot-tools
+RUN apt-get update && \
+	apt-get install -y \
+		wget \
+		gcc-arm-linux-gnueabi \
+		gcc-arm-linux-gnueabihf \
+		git \
+		build-essential \
+		bison \
+		flex \
+		bc \
+		u-boot-tools \
+		# to compile for 64bit ARM, particularly RPi3/4/5
```

Comment:
Thank you for the formatting and the comment. I suggest to remove the unused one (see above) and to add a comment for `gcc-arm-linux-gnueabi` that it is used for BBB

Reply:
Removed!

Reply:
I still think that a comment will be nice.